es/controllers/middleware.rst

```diff
@@ -1,6 +1,53 @@
 Middleware
 ##########
 
+Los objetos del middleware le dan la posibilidad de "envolver" su aplicación en capas compuestas y reutilizables
+de la lógica de manejo de la solicitud o respuesta. Visualmente, su aplicación termina en el centro y el 
+middleware se envuelve en la aplicación como una cebolla. Aquí podemos ver una aplicación con enrutamiento,
+activos, manejo de excepciones y middleware de encabezado CORS.
+
+.. image:: /_static/img/middleware-setup.png
```

Comment:
@markstory could we have these images/files in shared folder to share with all language ?
or using en version with  a something like  `../en/_static/img/middleware-setup.png` ?

Reply:
Shared assets are kind of a pain in how our sphinx is setup. For debugkit I ended up having to move assets into place with make. We could do that for the book as well, just takes doing.